datasets/cifar100/README.md

```diff
@@ -0,0 +1,149 @@
, 188, 85], [139, 175, 64], [168, 206, 88], [181, 213, 111], [173, 200, 100], [185, 212, 112], [204, 231, 131], [198, 225, 126], [151, 180, 84], [161, 192, 92], [127, 157, 62], [134, 161, 77], [167, 186, 105], [165, 176, 105], [163, 164, 104], [166, 161, 97], [201, 204, 133], [167, 164, 111], [145, 127, 100], [153, 134, 110], [150, 136, 108], [149, 137, 107], [134, 120, 81], [117, 100, 62], [100, 79, 47], [76, 57, 23], [83, 66, 35], [104, 89, 61]], [[164, 204, 83], [171, 211, 92], [182, 220, 110], [179, 216, 106], [156, 189, 96], [150, 185, 88], [165, 201, 94], [163, 196, 94], [171, 202, 98], [186, 217, 112], [202, 233, 128], [188, 219, 114], [173, 204, 98], [183, 214, 108], [185, 216, 110], [181, 213, 107], [174, 211, 98], [193, 229, 118], [184, 214, 109], [192, 219, 116], [204, 229, 124], [175, 183, 119], [154, 140, 125], [156, 138, 118], [150, 138, 107], [151, 141, 116], [143, 131, 97], [124, 109, 75], [105, 87, 60], [73, 56, 26], [81, 67, 30], [95, 85, 47]], [[153, 194, 65], [160, 202, 64], [158, 199, 69], [161, 200, 79], [157, 190, 97], [165, 198, 106], [155, 191, 83], [160, 195, 88], [172, 206, 97], [175, 209, 100], [173, 207, 98], [161, 195, 85], [161, 193, 84], [172, 202, 99], [175, 207, 100], [173, 205, 89], [175, 205, 90], [190, 219, 103], [188, 217, 101], [181, 209, 102], [179, 213, 97], [176, 198, 108], [155, 151, 110], [145, 131, 104], [140, 130, 101], [136, 130, 100], [129, 121, 83], [119, 108, 70], [77, 62, 33], [49, 34, 8], [70, 60, 17], [80, 75, 26]], [[148, 185, 79], [142, 182, 57], [140, 179, 60], [134, 173, 54], [140, 175, 76], [123, 157, 66], [109, 146, 39], [153, 191, 77], [145, 182, 69], [147, 183, 73], [145, 181, 71], [145, 179, 72], [112, 144, 45], [119, 150, 55], [135, 168, 64], [150, 185, 66], [165, 201, 72], [175, 213, 79], [183, 223, 89], [166, 205, 81], [156, 199, 75], [155, 189, 71], [147, 156, 76], [140, 131, 90], [138, 125, 98], [131, 128, 88], [124, 125, 69], [109, 103, 51], [43, 32, 4], [30, 17, 1], [65, 62, 15], [76, 77, 20]], [[122, 157, 66], [120, 155, 58], [126, 160, 71], [99, 135, 41], [93, 129, 36], [97, 133, 45], [113, 150, 53], [126, 166, 54], [129, 168, 55], [131, 168, 64], [141, 173, 76], [149, 179, 88], [116, 151, 67], [79, 115, 27], [105, 142, 41], [109, 148, 34], [114, 153, 32], [126, 163, 45], [137, 174, 60], [128, 165, 52], [131, 171, 50], [128, 170, 36], [140, 164, 64], [137, 133, 81], [137, 116, 81], [114, 112, 58], [115, 134, 45], [91, 91, 26], [30, 20, 2], [22, 16, 3], [97, 112, 56], [141, 161, 87]], [[87, 122, 41], [88, 122, 39], [101, 134, 56], [85, 117, 43], [72, 104, 30], [114, 148, 62], [146, 181, 81], [118, 155, 49], [131, 168, 66], [130, 163, 65], [151, 179, 90], [158, 184, 102], [148, 181, 89], [134, 169, 75], [126, 161, 65], [110, 145, 50], [99, 131, 48], [116, 149, 64], [125, 158, 67], [127, 160, 68], [116, 154, 48], [110, 151, 25], [112, 136, 43], [126, 122, 74], [133, 115, 76], [106, 114, 47], [113, 140, 47], [76, 78, 24], [19, 9, 4], [34, 36, 10], [105, 133, 59], [138, 173, 79]]]}
```

Comment:
You can remove the example, I think most people are not used to seeing images this way ;)